personal/privacyidea/PDF/class_RecoveryKeyPDF.inc

```diff
@@ -154,4 +154,18 @@
             $this->HREF='';
         }
     }
+
+    public function Salutation($sn, $givenname, $academictitle)
+    {
+        $this->SetFont('DejaVu', '', 10);
+        $greeter = "";
+
+        if (!empty($academictitle)) {
+            $greeter .= $academictitle . " ";
+        }
+
+        $greeter .= "$givenname $sn";
+        return $greeter;
+    }
+
```

Comment:
It would be good to leave the order and inclusion of the three parts and additional salutation formula to translators.

Reply:
This is a copy+paste from other code, we need to refactor this later.

Reply:
@sunweaver, please create an issue for this then. Thanks :)